configs/experiment/im2im/segmentation.yaml

```diff
@@ -34,3 +34,19 @@ data:
     # patch_shape: [64, 64]
     # 3D
     patch_shape: [16, 32, 32]
+
+callbacks:
+  # prediction
+  # saving:
+  #   _target_: cyto_dl.callbacks.image_saver.ImageSaver
+  #   save_dir: ${paths.output_dir}
+  #   save_every_n_epochs: ${model.save_images_every_n_epochs}
+  #   stages: ["predict"]
+  #   save_input: False
+  # training
+  saving:
```

Comment:
what is the saving key here? Shouldnt callbacks just be a list of callbacks?

Reply:
they all have a key : https://github.com/AllenCellModeling/cyto-dl/blob/323ff267fa061b3eace107ba44248677e878902c/configs/callbacks/default.yaml#L8